student.rb

```diff
@@ -10,6 +10,11 @@ def initialize(age:, classroom:, name: 'Unknown', parent_permission: true)
   def play_hooky
     '¯\\(ツ)/¯'
   end
+
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
```

Comment:
- [ ] good job initializing the `@classroom` 👍 however according to the requirements you still need to set up the getter attribute for this instance variable.

Reply:
- [ ] you did a great job on the project otherwise, this was really a minor comment. I hope everything will turn out ok with your personal issues.